api/services/GithubBuildStatusReporter.js

```diff
@@ -18,12 +18,14 @@ const checkAccessTokenPermissions = (users, site) => {
 
     return GitHub.checkPermissions(user, site.owner, site.repository)
     .then((permissions) => {
-      if (permissions.admin) {
+      if (permissions.push) {
         return Promise.resolve(user.githubAccessToken);
       }
-
       count += 1;
-
+      return getNextToken(users[count]);
+    })
+    .catch(() => {
+      count += 1;
```

Comment:
Should we log the catch error?

Reply:
i suppose we can ... it may help support conversations